src/components/ProtectionRefInput.tsx

```diff
@@ -129,7 +129,7 @@ export default function ProtectionRefInput<
       if (record?.id) {
         updateRecord(record.id as number, selectedData)
       } else if (id) {
-        createRecord(id, selectedData)
+        if (resource !== constants.R_ITEMS) createRecord(id, selectedData)
       }
```

Comment:
Hello @TahaKhanAbdalli - just to help my understanding... Can you teach me how this fixes the issue?

Reply:
My mistake, I was to stop redirecting only in the `useRefTable` hook if `resource = R_ITEMS`